settings_default.py

```diff
@@ -1,15 +1,15 @@
 from getpass import getpass
```

Comment:
Sorry, I forgot to remove that file, we don't use it anymore. The config is in `config.toml.example` and `config.toml`. Can you add an `agreements` section there?

Reply:
hah. yeah i updated my local config.toml, but forgot to put that in the example one. and my black run updated this file lol. Its deleted now tho